PyZMQ 25.0.0 pylint error about abstract classes

Because of [this commit](https://github.com/zeromq/pyzmq/commit/39102ac4672b2acda6542eb2a8ba2a459cf55b72), which was part of 25.0.0 pylint now complains about an abstract class being instantiated when using anything that inherits `AttributeSetter`, mainly `Context` & `Frame`.
Minimal example:
```python
import zmq
zmq.Context()
```
`# pylint zmq_test.py
************* Module zmq_test
zmq_test.py:3:0: E0110: Abstract class 'Context' with abstract methods instantiated (abstract-class-instantiated)`

--- a/zmq/sugar/attrsettr.py
+++ b/zmq/sugar/attrsettr.py
@@ -3,7 +3,6 @@
 # Copyright (C) PyZMQ Developers
 # Distributed under the terms of the Modified BSD License.
 
-import abc
 import errno
 from typing import TypeVar, Union
 
@@ -13,7 +12,7 @@ T = TypeVar("T")
 OptValT = Union[str, bytes, int]
 
 
-class AttributeSetter(metaclass=abc.ABCMeta):
+class AttributeSetter:
     def __setattr__(self, key: str, value: OptValT) -> None:
         """set zmq options by attribute"""
 
@@ -67,13 +66,13 @@ class AttributeSetter(metaclass=abc.ABCMeta):
         """override if getattr should do something other than call self.get"""
         return self.get(opt)
 
-    @abc.abstractmethod
     def get(self, opt: int) -> OptValT:
-        pass
+        """Override in subclass"""
+        raise NotImplementedError("override in subclass")
 
-    @abc.abstractmethod
     def set(self, opt: int, val: OptValT) -> None:
-        pass
+        """Override in subclass"""
+        raise NotImplementedError("override in subclass")
 
 
 __all__ = ['AttributeSetter']

